packages/google_cloud_gax/lib/gax.dart

```diff
@@ -14,59 +14,25 @@
 
 import 'dart:convert';
 
-import 'package:google_cloud_rpc/rpc.dart';
 import 'package:http/http.dart' as http;
-export 'src/web.dart'
-    if (dart.library.io) 'src/vm.dart'
-    show environmentVariable;
 
+import 'src/proto.dart';
```

Comment:
I like having a `proto.dart` file. Perhaps put 'Any' there, and 'Status' in a corresponding `rpc.dart` file? We could then document where they were from originally.

JsonEncodable, ProtoMessage, and ProtoEnum could then go in some other support file?

Reply:
I created:
- `proto.dart`
- `any.dart`
- `status.dart`